modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -232,6 +242,27 @@ The state of the left and right sidebars is now persisted. If you open the left
 * *Custom Permissions* +
 The custom permission selection `View`, `Edit`, `Create`, `Delete`, `Share` in sharing "Custom permissions" will no longer be available.
 
+== Infinite Scale 5.0.6 (Production)
+
+[discrete]
+=== General
+
+This is a patch release only, please update as soon as possible. +
```

Comment:
```suggestion
This is a patch release only, please update as soon as possible.
```

Reply:
Don't commit this,
the `+` symbol is intended to be placed here and part of the asciidoc language.
I can explain details if required 😄

Reply:
o_O wtf.
no details needed.